doc/monitoring/api_reference.rst

```diff
@@ -395,7 +395,7 @@ for details. To enable it you should register callback:
 
 **Prometheus query aggregated by thread name**
 
-.. code-block:: promql
+..  code-block:: text
```

Comment:
The PromQl lexer cannot parse this code example. I've checked it with other examples from Prometheus docs and it works fine with them. Maybe the code is a stub, maybe there's a syntax error.